src/js/page/ui/toasts.js

```diff
@@ -1,17 +1,21 @@
 import { strToEl, transitionToClass } from '../utils.js';
 
 class Toast {
-  constructor(message, duration, buttons) {
+  constructor(message, duration, buttons, isError) {
     this.container = strToEl(
       '<div class="toast"><div class="toast-content"></div></div>',
     );
-
     const content = this.container.querySelector('.toast-content');
-    content.textContent = message;
-
     this._answerResolve = null;
     this._hideTimeout = null;
 
+    if (isError) {
+      content.insertAdjacentHTML('afterbegin', '<pre><code></code></pre>');
```

Comment:
If there's a simpler way to do this, let me know.

Reply:
I'm happy with this approach